.github/BOTMETA.yml

```diff
@@ -694,7 +694,9 @@ files:
     labels:
       - networking
       - infoblox
-  $modules/net_tools/netbox/: FragmentedPacket
+  $modules/net_tools/netbox/:
+    maintainers: FragmentedPacket
+    migrated_to: netbox_community.ansible_modules
```

Comment:
`migrated_to` needs to be against every file, not the directory. See other examples in this file.

Reply:
This has been changed.